apps/src/templates/sectionProgressV2/progress-key-popup.module.scss

```diff
@@ -0,0 +1,78 @@
+@import 'color.scss';
+@import 'font';
+
+.item {
+  display: inline-flex;
+  gap: 12px;
+  margin-top: 20px;
+
+  p {
+    margin-bottom: 0;
+    font-size: 0.875rem;
+  }
+
+  .firstMarkdown {
+    margin-bottom: 1.25em;
+  }
+}
+
+.dialog {
+  margin-left: 8px;
+  margin-right: 8px;
+
+  hr {
+    margin-top: 0;
+  }
+
+  h6:not(:first-of-type) {
+    margin-top: 40px;
+  }
+
+  h6 {
+    margin-bottom: 0;
+  }
+}
+
+.xCloseButton {
+  // Clear default button styles
+  background: none;
+  color: inherit;
+  border: none;
+  padding: 0;
+  font: inherit;
+  cursor: pointer;
+  outline: inherit;
+  box-shadow: none;
+
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+
+  i {
```

Comment:
Does this need to be in the `i` or can it just be in `.xCloseButton`?

Reply:
I tried moving it outside and it does change the appearance of the button when it is hovered over... I am still reading into why.
<img width="43" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/24883357/3f636bfe-0ce3-4f8c-8502-5b6db1b75c80">